rmf_task_ros2/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ include(GNUInstallDirs)
 
 find_package(ament_cmake REQUIRED)
 find_package(backward_ros REQUIRED)
+find_package(std_msgs REQUIRED)
```

Comment:
This was an interesting change, cleaning up the `geometry_msgs` dependencies where they were not needed upstream removed a transitive dependency on `std_msgs` for this package that made it fail compiling because [this std_msgs header](https://github.com/open-rmf/rmf_ros2/blob/8dee3afeb9513be711b8f1c2dea9338b57bbdeae/rmf_task_ros2/src/rmf_task_ros2/Dispatcher.cpp#L49) was not found.
Since the package needs std_msgs it should include it in its package.xml and CMakeLists file explicitly rather than rely on transitive dependencies